lib/ahoy_email/processor.rb

```diff
@@ -159,11 +149,8 @@ def parse_uri(href)
       Addressable::URI.parse(href.to_s) rescue nil
     end
 
-    def url_for(opt)
-      opt = (ActionMailer::Base.default_url_options || {})
-            .merge(options[:url_options])
-            .merge(opt)
-      AhoyEmail::Engine.routes.url_helpers.url_for(opt)
+    def default_url
+      @opt ||= AhoyEmail::Engine.routes.url_helpers.url_for((ActionMailer::Base.default_url_options || {}).merge(controller: "ahoy/messages", action: "click", id: "$ID$"))
```

Comment:
It's a good practice to name the instance variable the same as the method - i.e. `@default_url`.